### ISSUE TITLE
Add an option to un-ignore bots

### ISSUE THREAD

**Issue by rpdelaney** [nedbat/dinghy#25]
For instance, I'm interested in ancient dependabot PRs because that usually means CI is broken on main. We're also migrating from dependabot to renovatebot, so it would be useful if I could un-ignore specific bots (i.e. to find repos where dependabot isn't fully gone).

**Comment by rpdelaney** [comment]
There are different ways of achieving this. Off the top, you might create a configuration keyword to un-ignore specific users. If used, it would over-ride the bot-ignoring behavior. For instance, something like `ignore_users: NOT dependabot NOT renovatebot`. That wouldn't break existing configurations, and it would satisfy my user story.

You could also create a configuration keyword that means "all bots" so that a user who wants every bot to show up could do `ignore_users: NOT %bots%`, or `ignore_users: %bots% NOT renovatebot` etc.

**Comment by nedbat** [comment]
I added an `include_bots: true` setting that I think does what you want:
```
pip install git+https://github.com/nedbat/dinghy.git@nedbat/include-bots
```
Let me know if it seems right to you.  I wonder if the bot users should be indicated somehow in the output?

**Comment by rpdelaney** [comment]
Yeah, that seems to do the thing.

> I wonder if the bot users should be indicated somehow in the output?

It would be useful if I could group them separately. Maybe just `bots: true` means I get _only_ bots? Then I could do this:

```yaml
---
digests:
  - digest: humans.html
    since: forever
    title: Trussworks Humans
    items:
      - search: org:trussworks is:open archived:false no:assignee    type:pr
        title: Pull requests
  - digest: bots.html
    since: forever
    title: Trussworks Bots
    bots: true
    items:
      - search: org:trussworks is:open archived:false no:assignee    type:pr
        title: Pull requests
```

**Comment by nedbat** [comment]
I want to merge this as-is, and we can talk about `bots: true` (or whatever) in another place.

**Comment by nedbat** [comment]
This is finished in commit e732d95.

**Comment by nedbat** [comment]
This is now released as part of [dinghy 0.15.0](https://pypi.org/project/dinghy/0.15.0).
